book/src/ephemeral-local-volumes.md

```diff
@@ -2,15 +2,44 @@
 
 ## Status
 
-Status | Min K8s Version | Max K8s Version 
+### CSI Ephemeral Inline Volumes
+
+Status | Min K8s Version | Max K8s Version
 --|--|--
 Alpha | 1.15 | 1.15
-Beta | 1.16 | -
+Beta | >=1.16 | -
+
+### Generic Ephemeral Inline Volumes
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.19 | -
 
 ## Overview
-Traditionally, volumes that are backed by CSI drivers can only be used with a `PersistentVolume` and `PersistentVolumeClaim` object combination. This feature supports ephemeral storage use cases and allows CSI volumes to be specified directly in the pod specification.  At runtime, nested inline volumes follow the ephemeral lifecycle of their associated pods where the driver handles all phases of volume operations as pods are created and destroyed.
+Traditionally, volumes that are backed by CSI drivers can only be used with a `PersistentVolume` and `PersistentVolumeClaim` object combination. The "CSI Ephemeral Inline volumes" feature allows CSI volumes to be specified directly in the pod specification for ephemeral use cases.  At runtime, nested inline volumes follow the ephemeral lifecycle of their associated pods where the driver handles all phases of volume operations as pods are created and destroyed.
+
+CSI drivers have to be modified to support "*CSI* Ephemeral Inline
```

Comment:
What if we make this a new section, "Which feature should my driver support?", and go into details on some use cases and maybe examples.

Reply:
I've added that. Please take another look.